README.md

```diff
@@ -1 +1,8 @@
 # ocm-common
+This project contains code which was duplicated between clients/services. The goal of this project is to simplify our codebases by removing duplicate code from our codebases.
```

Comment:
NP : `This project contains utility functions intended for sharing across OCM clients and services. Its objective is to streamline OCM codebases by eliminating duplicate code found within them.`